manifests-servicemesh/1.2.0/servicemeshoperator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -597,9 +584,6 @@ spec:
                 - name: operator-olm-config
                   mountPath: /etc/operator/olm
                   readOnly: true
-                - name: webhook-tls-volume
```

Comment:
I think these might have been missed when you split out operator.yaml.

Reply:
Ugh, yeah, thanks! Looks like I didn't notice these changes when I rebased today.  

I've now reviewed the files again to make sure nothing else has disappeared.